scipy/stats/tests/test_distributions.py

```diff
@@ -518,6 +518,34 @@ def test_gennorm(self):
         assert_almost_equal(pdf1, 2*pdf2)
 
 
+class TestLaplaceasymmetric(object):
+    def test_laplace(self):
+        # test against Laplace (special case for kappa=1)
+        points = np.array([1, 2, 3])
+        pdf1 = stats.laplace_asymmetric.pdf(points, 1)
+        pdf2 = stats.laplace.pdf(points)
+        assert_almost_equal(pdf1, pdf2)
```

Comment:
```suggestion
        assert_allclose(pdf1, pdf2)
```
[NumPy suggests](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html) transitioning to `assert_allclose`.

Reply:
Done thanks.